src/clr-angular/forms/common/form.ts

```diff
@@ -28,4 +52,60 @@ export class ClrForm {
   markAsTouched() {
     this.markControlService.markAsTouched();
   }
+
+  @ContentChildren(ClrLabel, { descendants: true })
+  labels: QueryList<ClrLabel>;
+
+  @HostListener('submit')
+  onFormSubmit() {
```

Comment:
I updated the hostlistener to call markAsTouched() and added optional args and a guard in t markAsTouched that allow consumers to prevent the aria live from being updated.